package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "client",
+  "homepage": "http://LibertyDSNP.github.io/example-client",
```

Comment:
We still need a blockchain backend for this to work. Currently we've just been using the old demo contracts repo, but we can probably work on something new. Perhaps we can even include that in the repo? We should probably discuss it next week.